data_processing/raw_data_sanity.py

```diff
@@ -146,24 +146,44 @@ def read_sentinel5p(subsubfolder, data_key):
 print("2021 shape: ", df_2021.shape)
 # -
 
+len(df_2021.time_utc.unique())
+
 # CONCATENATING ALL DATA
 df_combined = df_2018
 df_combined = pd.concat([df_combined,df_2019], ignore_index=True)
 df_combined = pd.concat([df_combined,df_2020], ignore_index=True)
 df_combined = pd.concat([df_combined,df_2021], ignore_index=True)
 print("Total shape: ", df_combined.shape)
 
+df_combined.head()
+
+df_combined
+
+groupby_date
+
 # +
 df_combined['date'] = pd.to_datetime(df_combined['time_utc']).dt.date
-groupby_date = df_combined.groupby(df_combined.date).count()
+groupby_date = df_combined.groupby(df_combined['date']).count()
 
 plt.figure(figsize=(40,20))
-plt.bar(df_combined.date.unique(), groupby_date.time_utc)
+plt.bar(groupby_date.index, groupby_date.time_utc)
 # -
 
-groupby_date
+groupby_date.time_utc.describe()
 
+plt.boxplot(groupby_date.time_utc)
 
+# +
+#Write the dataframe to 1 parquet file
+file_name='combined-raw.parquet'
+
+bucket = 'methane-capstone'
+subfolder = 'combined-raw-data'
+s3_path_month = bucket+'/'+subfolder
+
+df_combined.to_parquet('s3://{}/{}'.format(s3_path_month,file_name))
```

Comment:
Can we write the data as **`.parquet.gzip`**

Reply:
yes. will update this later